src/components/_inputs/InputDiscount.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <fieldset>
+    <legend>{{schema.title}}</legend>
+    <a-input-group size="large">
+      <a-form-item :label="type.title">
+        <a-select v-model="data.type">
+          <a-select-option v-for="item in type.enum" :value="item">
+            {{item}}
+          </a-select-option>
+        </a-select>
+      </a-form-item>
```

Comment:
O `v-for` precisa ter `key`, mas não vou sugerir esta alteração porque acho que na verdade aqui você deveria usar o `InputEnum`, certo?